resources/views/themes/default1/front/checkout.blade.php

```diff
@@ -165,49 +165,51 @@
                 <h4 class="heading-primary">Payment</h4>
              
                        
-                {!! Form::open(['url'=>'checkout','method'=>'post']) !!}
+                {!! Form::open(['url'=>'checkout','method'=>'post','id' => 'checkoutsubmitform' ]) !!}
 
                 @if(Cart::getTotal()>0)
                   
                  <?php 
                 $gateways = \App\Http\Controllers\Common\SettingsController::checkPaymentGateway($item['attributes']['currency']['currency']);
                 $total = Cart::getSubTotal();
                 $rzpstatus = \App\Model\Common\StatusSetting::first()->value('rzp_status');
+
                   // 
                 ?>
                 @if(count($gateways)>0 ) 
-                  <div class="form-group">
+                <div class="row">
+                  
 
                     <div class="col-md-6">
                         @foreach($gateways as $gateway)
-                        {{ucfirst($gateway)}} {!! Form::radio('payment_gateway',strtolower($gateway)) !!}<br><br>
+                        <?php
+                        $processingFee = \DB::table(strtolower($gateway))->where('currencies',$item['attributes']['currency']['currency'])->value('processing_fee');
+                        ?>
+                        {!! Form::radio('payment_gateway',$gateway,false,['id'=>'allow_gateway','data-currency'=>$processingFee]) !!}
+                         <img alt="{{$gateway}}" width="111"  src="{{asset('client/images/'.$gateway.'.png')}}">
+                          <br><br>
+                       <div id="fee" style="display:none"><p>An extra processing fee of <b>{{$processingFee}}%</b> will be charged on your Order Total during the time of payment</p></div>
```

Comment:
why is this div hidden?

Reply:
We only show this div when Stripe payment radio button is checked